pkgs/top-level/all-packages.nix

```diff
@@ -20473,8 +20473,6 @@ in
 
   kdeconnect = libsForQt5.callPackage ../applications/misc/kdeconnect { };
 
-  kdecoration-viewer = libsForQt5.callPackage ../tools/misc/kdecoration-viewer { };
```

Comment:
please add a `throw` in aliases for deprecation

Reply:
Thanks! I forgot to commit that line.